quickstart_template/101-spinnaker-aks.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+
+function print_usage() {
+  cat <<EOF
+Command
+  $0 
+
+Arguments
+  --app_id|-ai                           [Required]: Service principal app id used by Spinnaker to dynamically manage resources
+  --app_key|-ak                          [Required]: Service principal app key used by Spinnaker to dynamically manage resources
+  --username|-u                          [Required]: vm username
+  --password|-p                          [Required]: vm password
+  --tenant_id|-ti                        [Required]: Tenant id
+  --subscription_id|-si                  [Required]: Subscription id
+  --resource_group|-rg                   [Required]: Resource group containing your key vault and packer storage account
+  --vault_name|-vn                       [Required]: Vault used to store default Username/Password for deployed VMSS
+  --storage_account_name|-san            [Required]: Storage account name used for front50
+  --storage_account_key|-sak             [Required]: Storage account key used for front50
+  --aksClusterName|-aks                  [Required]: aksClusterName for deploy spinnaker
```

Comment:
As for the AKS name, suggest to use lower case and underline like other arguments. E.g.:  aks_cluster_name | -can. Please note also change following occurrences of this argument in this file.